includes/base/widget-base.php

```diff
@@ -774,4 +774,34 @@ public function remove_skin( $skin_id ) {
 	public function get_skins() {
 		return Plugin::$instance->skins_manager->get_skins( $this );
 	}
+
+	/**
+	 * @param string $plugin_title  Elementor-Pro | Elementor-Core
+	 * @param string $since         Plugin version widget was deprecated
+	 * @param string $last          Plugin version in which the widget will be removed
+	 * @param string $replacement   Widget replacement
+	 */
+	protected function deprecated_notice( $plugin_title, $since, $last, $replacement ) {
```

Comment:
`$last, $replacement` should be optionally.

Reply:
done